frontend/src/app/update-file/update-file.component.ts

```diff
@@ -29,6 +45,7 @@ export class UpdateFileComponent implements OnChanges {
   updateTags: string[] = [];
   uploadInProgress: boolean = false;
   showHint: boolean = false;
+  isUpdating: boolean = false;
 
   @ViewChild('fileInput') fileInput!: ElementRef;
   @Input() updateItem!: any;
```

Comment:
There's still an any here that needs to be changed.

Reply:
fixed in the latest commit https://github.com/GoogleCloudPlatform/app-large-data-sharing-golang/pull/1/commits/6b656573d3db05f371011f3a0ae3f416e9b4fc74